presto-main/src/main/java/io/prestosql/server/security/jwt/JwkService.java

```diff
@@ -53,6 +54,12 @@
     @Generated("this")
     private Closer closer;
 
+    @Inject
+    public JwkService(JwtAuthenticatorConfig config, @ForJwk HttpClient httpClient)
+    {
+        this(URI.create(config.getKeyFile()), httpClient, new Duration(15, TimeUnit.MINUTES));
```

Comment:
why it named a file? maybe remote file?

Reply:
I am reusing the existing config properties.  So if you set the key file to a URL, we switch from actual files to JWK